functests/2_performance_update/updating_profile.go

```diff
@@ -144,6 +144,28 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			}
 		})
 
+		It("[test_id:22764] verify that realTimeKernel is default when realTimeKernel field does not exist", func() {
+			conditionUpdating := machineconfigv1.MachineConfigPoolUpdating
+
+			if profile.Spec.RealTimeKernel == nil || *profile.Spec.RealTimeKernel.Enabled == true {
+				Skip("Skipping test - This test expects RT Kernel to be disabled. Found it to be enabled or nil.")
+			}
+
+			By("Applying changes in performance profile")
+			profile.Spec.RealTimeKernel = nil
+			Expect(testclient.Client.Update(context.TODO(), profile)).ToNot(HaveOccurred())
+
+			Expect(profile.Spec.RealTimeKernel).To(BeNil())
+			By("Checking that the updating MCP status will consistently stay false")
+			Consistently(func() corev1.ConditionStatus {
+				return getMcpStatus(testutils.RoleWorker, conditionUpdating)
```

Comment:
I think the should be `testutils.RoleWorkerRT`?
(on my cluster the worker MCP was updating for another reason, which is a funny coincidence, because otherwise I would not have spotted this one)

Reply:
Yes, that should be testutils.RoleWorkerRT